VMDIRAC/Resources/Cloud/OpenStackEndpoint.py

```diff
@@ -206,7 +206,10 @@ def createInstance(self, instanceID=''):
       flavorID = self.flavors.get(flavor)["FlavorID"]
       if not flavorID:
         return S_ERROR('Can not get ID for the flavor: %s' % flavor)
+      numberOfProcessors = self.flavors.get(flavor)["NumberOfProcessors"]
     self.parameters['FlavorID'] = flavorID
+    if "NumberOfProcessors" not in self.parameters:
+      self.parameters["NumberOfProcessors"] = numberOfProcessors
```

Comment:
I know pretty much nothing about VMDIRAC, so forgive me if I write silly stuff, but...
IIUC this "NumberOfProcessors" is the number of processors of the VMs that VMDIRAC will try to instantiate. I see this is going in the OpenStack endpoint module, but I think it should be common to all the endpoints, so going in https://github.com/DIRACGrid/VMDIRAC/blob/integration/VMDIRAC/Resources/Cloud/CloudEndpoint.py ?